content/en/user-manual/scripting/migration-guide.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Migration Guide
+template: usermanual-page.tmpl.html
+position: 10
+---
+
+## Migrating Legacy Script Projects
+
+In July 2016, PlayCanvas adopted its [current scripting system][1] (sometimes known as Scripts 2.0). Legacy script projects continue to work as normal. However, it is no longer possible to fork or create new legacy script projects.
+
+In December 2020, it was announced that legacy script projects would be made read-only in the near future. When you open one in the Editor, you will see the following message in the project dashboard:
+
+![Dashboard Legacy Script][2]
+
+And the following message in the Editor:
+
+![Editor Legacy Script][3]
+
+If you want to continue to work on legacy script projects, we recommend that you migrate them to the current format. Unfortunately, there is no automated migration process for this. Instead, it is a manual process, but this guide will walk you through it.
+
+### Step 1 - Create a New Project
+
+We will begin by creating a new, blank project to receive the update project. Delete the four entities created by default below the scene root entity.
+
+### Step 2 - Transfer Assets
+
+Next, copy all assets to your new project (minus scripts which in legacy projects are *not* true assets). It is possible to copy and paste assets from one project to another (using the right-click context menu).
+
+![Copy Paste Assets][4]
+
+However, legacy script projects are quite old. Years ago, assets were imported without file extensions and some metadata might not be available to view in the Inspector panel. So you might want to consider downloading your source assets and uploading them to your new project.
+
+### Step 3 - Transfer Scripts
+
+Download the scripts from your legacy script project via the Project Dashboard:
+
+![Download Scripts][5]
+
+If your legacy project is connected to a GitHub repo, download your scripts from there.
+
+You can now upload the legacy scripts to your new project. 
```

Comment:
Maybe show a screenshot of upload process in the assets panel.

Reply:
Not sure how to do this unless it's an animated GIF. I'd hope devs would know how to upload assets though (if they have legacy projects).